### PR TITLE
Follow on for Store parsed CIDRs at initialization of Proxier #76779

### PR DESCRIPTION
**Pull request by tedyu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR addresses review comments from #76779.

For err checking in parseExcludedCIDRs, I think the current form should be fine since it has 3 lines while the suggested alternative has 4 lines.

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/ipvs/proxier.go [hunk 1]
```diff
@@ -274,11 +274,11 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
...
-// ParseExcludedCIDRs parses the input strings and returns net.IPNet
+// parseExcludedCIDRs parses the input strings and returns net.IPNet
 // The validation has been done earlier so the error condition will never happen under normal conditions
-func ParseExcludedCIDRs(excludeCIDRStrs []string) []*net.IPNet {
+func parseExcludedCIDRs(excludeCIDRs []string) []*net.IPNet {
 	var cidrExclusions []*net.IPNet
-	for _, excludedCIDR := range excludeCIDRStrs {
+	for _, excludedCIDR := range excludeCIDRs {
 		_, n, err := net.ParseCIDR(excludedCIDR)
 		if err == nil {
```
Comment (andrewsykim) [inline pkg/proxy/ipvs/proxier.go]: Do you think it still makes sense to flip the err check here and add logging if ParseCIDR fails?
```go
if err != nil {
     klog.Errorf("Error parsing exclude CIDR %q,  err: %v", excludedCIDR, err)
     continue
}

cidrExclusions = append(cidrExclusions, n)
```

Comment (tedyu) [reply pkg/proxy/ipvs/proxier.go]: I have changed the code as suggested.

**Comment by andrewsykim** [comment]
/lgtm
/assign @m1093782566 
for approval

**Comment by m1093782566** [comment]
/approve

Thanks!

**Pull request merged**